catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/ExportCommand.java

```diff
@@ -176,6 +177,47 @@ protected Object executeWithSubject() throws Exception {
     revisionFilter = initRevisionFilter();
 
     final File outputFile = initOutputFile(output);
+    checkFile(outputFile);
+
+    if (delete && !force) {
+      final String input =
+          session.readLine(
+              "This action will remove all exported metacards and content from the catalog. Are you sure you wish to continue? (y/N):",
+              null);
+      if (!input.matches("^[yY][eE]?[sS]?$")) {
```

Comment:
I agree that matching to yE is weird when not just checking for the first character. 
Is there another reason regex is bad here? 
For would it be a bad idea to instead have  `!input.matches("^[yY]")`?

Reply:
Well, that wouldn't match `yes`, so that's a problem. 😄 
You'd need to change it to `^[y|Y].*`

RegEx is pretty heavy, even in a small case like this, when all you really care about is the value of the first character and can do an equality match.